tests/test_benchmarks.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from .utils import *
+
+TESTDIR = os.path.dirname(__file__)
+
+class BenchmarkTests(TranspileTestCase):
+
+    def test_binary_trees(self):
+        with open(os.path.join(TESTDIR + "/benchmarks/", "binary-trees.py")) as javafile:
+            out = self.runAsJava(adjust(javafile.read()), args=["5", "5"], timed=True)
+
```

Comment:
At column 34: (F405) 'adjust' may be undefined, or defined from star imports: .utils